app/models/patient.rb

```diff
@@ -10,6 +10,29 @@ class Patient < ApplicationRecord
   include ActiveModel::Validations
   include FhirHelper
 
+  MONITORING_REASONS = [
+    'Not Monitoring',
+    'Completed Monitoring',
+    'Meets criteria to shorten quarantine',
+    'Does not meet criteria for monitoring',
+    'Enrolled more than 14 days after last date of exposure (system)',
```

Comment:
do we want the system ones available in the dropdown?

Reply:
@kylietmo any preference? I copied them from the master list from the `validation_helper`

Reply:
This PR should not cause a functionality change and it wouldn't make sense for users to be able to select system-specific options, so no those should not be included in the UI dropdowns. It should work exactly as it did previously, and this should just be code consolidation.